.github/workflows/macos.yml

```diff
@@ -0,0 +1,68 @@
+name: macOS
+
+on: [push, pull_request]
+
+jobs:
+  macOS:
+    runs-on: macOS-10.15
```

Comment:
To switch, we've had to bump from 10.14 to 10.15. But we are still setting `MACOSX_DEPLOYMENT_TARGET: "10.11"`.